completion/grruby.cwl

```diff
@@ -0,0 +1,34 @@
+# grruby package
+# ponte-vecchio 2023-01-17 for v0.7
+
+#keyvals:\usepackage/grruby#c
+rubystyle=#default,gromob,XXruby,ruby,progress,rup,lw,rw,user
+rubycolor=%<color%>#%color
+expos=#true,false
```

Comment:
For boolean keys you should just write `expos#true,false`

Reply:
Does this work for when the default is `expos=false`, and you must pass `expos=true` in order to enable some features cf. just passing `expos` to assume `true`?

Reply:
Almost every way to define boolean keys in latex views `expos` and `expos=true` the same. That's true here since it's done with l3keys. So the special `expos#true,false` makes it such that the completer will only complete the key to `expos`, not `expos=` as it would with `expos=#true,false`. The only time it makes sense to write `key=#true,false` is when the default is true, which IMO is bad practice on the part of the package writer

Reply:
Resolved! just went with `expos#true,false`